pkg/providers/instance/skuclient/skuclient.go

```diff
@@ -48,11 +50,14 @@ func (sc *skuClient) updateInstance() {
 	sc.mu.RLock()
 	defer sc.mu.RUnlock()
 
-	authorizer, err := auth.NewAuthorizer(sc.cfg, sc.env)
+	// Create a new authorizer for the sku client
+	// TODO (charliedmcb): need to get track 2 support for the skewer API
+	cred, err := azidentity.NewDefaultAzureCredential(nil)
```

Comment:
How did you test this?

Reply:
It is the code of `auth.NewAuthorizer(sc.cfg, sc.env)`, with changes to use `azidentity.NewDefaultAzureCredential(nil)`.
E2E for each auth method should be sufficient.